src/Controller/RegistrationController.php

```diff
@@ -123,4 +123,10 @@ public function resendVerificationEmail(Request $request, SessionInterface $sess
 
         return $this->render('site/registration/resend_verification.html.twig');
     }
+
+    #[Route('/register/by/invitation', name: 'site.register.by.invitation')]
```

Comment:
do you have a better idea of route name ?

Reply:
Nope